templates/validator.py

```diff
@@ -57,7 +57,7 @@
         S3Bucket=Ref(param_s3_bucket_name),
         S3Key=Ref(param_s3_key),
     ),
-    Runtime='nodejs12.x',
+    Runtime='nodejs14.x',
     Handler='index.handler',
     Role=Ref(param_role),
     Tags=GetAtt(cloudformation_tags, 'TagList'),
```

Comment:
Ik denk dat je hier ook een Env Var wilt zetten (en op de andere functies ook) met

`POWERTOOLS_SERVICE_NAME="authorizer"`

niet zo belangrijk voor ons (we hebben maar 1 service), maar ik  weet niet wat anders de default is